tests/installation/validation/ibft.pm

```diff
@@ -117,6 +118,9 @@ sub run {
     $ibft_expected->{target}->{ip_addr} = inet_ntoa(inet_aton($fqdn));
 
     select_console 'root-console';
+    # Workaround for packages of lsscsi and hdparm are not pre-installed on sle16
+    zypper_call 'in lsscsi hdparm' if is_sle('>=16');
```

Comment:
please check this again with latest build.

Reply:
the bugs were fixed, and removed the workaround, thanks.